### ISSUE TITLE
Running MuSE call goes segmentation fault on a small sample

### ISSUE THREAD

**Issue by jumpyknight** [danielfan/MuSE#4]
I'm running MuSE on a very small testing sample, which is about 1Mbp in length. 
My command is
    MuSE call -f ref.fa tumor.bam normal.bam -O test
and the output is segmentation fault after a few seconds, and an unfinished .txt file.
I tried to add the -l or -r argument in my command but the output remains the same.
I have already tested this sample on a few other somatic callers, so the sample should be fine.
Does anyone have met the same problem?

**Comment by mikisvaz** [comment]
Yes, I get a segmentation fault as well for several samples I have tested.

**Comment by eriicdesousa** [comment]
I got the same problem when compiling the version from github but it worked fine when I used this binary http://bioinformatics.mdanderson.org/Software/MuSE/MuSEv1.0rc_submission_c039ffa
